src/util/templateValidation.ts

```diff
@@ -12,6 +12,16 @@ export function toSafeName(name: string) {
     return name.trim().replaceAll(" ", "_").toLowerCase();
 }
 
+/**
+ * Like {@link toSafeName}, but returns `undefined` if {@link name} is `undefined`.
+ * 
+ * @param name The name to convert.
+ * @returns The safe name if {@link name} is a string, otherwise `undefined`.
+ */
+export function toSafeNameSafe(name: string | undefined) {
```

Comment:
Shouldn't this replace the function above? Otherwise, name the function above `toSafeString` (and don't export it) and name this `ensureSafeString.

Reply:
`toSafeString` is used in `artistCTToInput` and other `CTToInput`s that have a `safeName` field since `name` is required for creation, so exporting is required

Reply:
renamed